Makefile

```diff
@@ -59,6 +59,12 @@ ROCKSDB_SYS_PORTABLE=0
 RUST_TEST_THREADS ?= 2
 endif
 
+# Enable flags for AWS Graviton2
+ifeq ($(shell uname -p),aarch64)
+RUSTFLAGS="-Ctarget-cpu=native -C target-feature=+lse"
```

Comment:
can you hide this behind an env var? We don't want to enable cpu specific flags by default.